Tests/Adyen Tests/UI/Form/FormItems/Address/FormAddressItemTests.swift

```diff
@@ -0,0 +1,26 @@
+//
+// Copyright (c) 2023 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+@_spi(AdyenInternal) @testable import Adyen
+import XCTest
+
+class FormAddressItemTests: XCTestCase {
+
+    func testCountrySelectItemUpdate() throws {
+        
+        let formAddressItem = FormAddressItem(
+            initialCountry: "NL",
+            style: .init(),
+            supportedCountryCodes: ["NL", "US"],
+            addressViewModelBuilder: DefaultAddressViewModelBuilder()
+        )
+        
+        XCTAssertEqual(formAddressItem.countrySelectItem.value.identifier, "NL")
+        
+        formAddressItem.value = .init(country: "US")
+        XCTAssertEqual(formAddressItem.countrySelectItem.value.identifier, "US")
```

Comment:
let's add cases where address is not in the supported country codes

Reply:
Good one! Fixed in 0df5fb50054af34dd612bfdd0065bdd5a587ac08